examples/sinatra/exporter.rb

```diff
@@ -0,0 +1,15 @@
+require "new_relic/telemetry_sdk"
+
+class Exporter
```

Comment:
Let's remove this - we will be working on an OpenTelemetry exporter soon, which will be a good real-life example. It's a bit difficult to make an example exporter that is realistic yet simple enough to be digestible, IMO.